packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -113,13 +114,30 @@ class TimelineScreen extends Screen {
     serviceManager.onConnectionClosed.listen(_handleConnectionStop);
 
     framesBarChart.onSelectedFrame.listen((TimelineFrame frame) {
-      frameDetailsContainer.attribute('hidden', frame == null);
-
       if (frame != null && timelineController.hasStarted) {
-        frameFlameChart.updateFrameData(frame);
+        flameChart.attribute('hidden', frame == null);
+        eventDetails.attribute('hidden', frame == null);
+
+        flameChart.updateFrameData(frame);
+        eventDetails.reset();
+
+        // Configure the flame chart / event details splitter if we haven't
+        // already.
+        if (!splitterConfigured) {
```

Comment:
Perhaps just build and configure the splitter with the rest of the UI? Is the lazy construction so that the splitter doesn't show until there's content in the timeline view?

Reply:
Yeah, I thought it looked a little strange with the splitter being visible when the content it is splitting is not.